aldryn_people/admin.py

```diff
@@ -61,8 +65,7 @@ def formfield_for_foreignkey(self, db_field, request=None, **kwargs):
     fieldsets = (
         (None, {
             'fields': (
-                'name',
-                'slug',
+                ('name', 'slug', ),
```

Comment:
@mkoistinen We decided to place the name and slug fields under each other and make them full width because the slug field was too small before.